dhall/src/Dhall/Main.hs

```diff
@@ -15,16 +17,20 @@ module Dhall.Main
     , ResolveMode(..)
     , parseOptions
     , parserInfoOptions
+    , findConfig
+    , resolveSettings
 
       -- * Execution
     , Dhall.Main.command
     , main
+    , CharacterSet(..)
```

Comment:
I exported this so I could refer to it in the tests – a bit awkward, since CharacterSet is defined in Internal.hs